lib/pharos/configuration/host.rb

```diff
@@ -18,23 +19,85 @@ class Host < Pharos::Configuration::Struct
       attribute :taints, Pharos::Types::Strict::Array.of(Pharos::Configuration::Taint)
       attribute :user, Pharos::Types::Strict::String
       attribute :ssh_key_path, Pharos::Types::Strict::String
+      attribute :ssh_port, Pharos::Types::Strict::Integer.default(22)
       attribute :ssh_proxy_command, Pharos::Types::Strict::String
       attribute :container_runtime, Pharos::Types::Strict::String.default('docker')
       attribute :environment, Pharos::Types::Strict::Hash
-      attribute :bastion, Pharos::Configuration::Bastion
+      attribute :bastion, Pharos::Configuration::Bastion.optional.default(nil)
 
       attr_accessor :os_release, :cpu_arch, :hostname, :api_endpoint, :private_interface_address, :resolvconf, :routes, :config
 
+      # @return [String]
       def to_s
         short_hostname || address
       end
 
+      # @return [String,nil]
       def short_hostname
         return nil unless hostname
 
         hostname.split('.').first
       end
 
+      # @return [Hash]
+      def ssh_options
```

Comment:
IMO this should be under `Transport` (for example `Transport::SshHelpers` module)